joomla/cassiopeia
# Cassiopeia/adjust toggle google font

Pull Request for Issue # .

### Summary of Changes

Current implementation to toggle the use of Google Fonts differs from the other toggles on tab Advanced of the Cassiopeia template. This PR solves this by adjusting the field parameters. 

### Testing Instructions

- Checkout branch development
- Login to backend.
- Edit the Cassiopeia template style and check the "Advanced" Tab.
- Notice the layout of the _Google Fonts or Text and Heading_ part
- Apply this PR and refresh the page. 
- Notice the layout of the _Google Fonts or Text and Heading_ part

### Expected result

<img width="915" alt="Schermafbeelding 2020-09-07 om 09 08 31" src="https://user-images.githubusercontent.com/639822/92358665-227f4100-f0ea-11ea-85b7-1130ef30ffb1.png">


### Actual result

<img width="914" alt="Schermafbeelding 2020-09-07 om 09 08 15" src="https://user-images.githubusercontent.com/639822/92358660-20b57d80-f0ea-11ea-88ef-3eaa145f08f6.png">


### Documentation Changes Required
